medusa/providers/torrent/html/gftracker.py

```diff
@@ -175,14 +174,15 @@ def process_column_header(td):
 
                     torrent_size = cells[labels.index('Size/Snatched')].get_text(strip=True).split('/', 1)[0]
                     size = convert_size(torrent_size, units=units) or -1
-
+                    pubdate = cells[labels.index('Added')].get_text(" ")
+                    
```

Comment:
its not about that. need to be a datime object. medusa converts datetime object in generic_provider, if is not, it will fail